compilers/concrete-compiler/compiler/src/main.cpp

```diff
@@ -119,6 +119,12 @@ llvm::cl::opt<bool> emitGPUOps(
         "enable/disable generating GPU operations (Disabled by default)"),
     llvm::cl::init<bool>(false));
 
+llvm::cl::opt<bool>
```

Comment:
do you want more tests before making it the default ?

Reply:
so we don't want to make it as default as is actually a tradeoff, between bandwidth space and latency. We could decide later to make it default.